tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-Upgrade.md

```diff
@@ -1,39 +1,227 @@
 * None
+
+
+Test - Upgrade from OVA 1.1.1 to 1.2.0
+======================================
+**[Deploy Old OVA Link]** [Deploy Old OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.1/vic_vsphere_admin/deploy_vic_appliance.html)
+
+**[OVA Old Version]** 1.1.1
+
+**[Root Cert Path]** Admin > Download Root Cert
+
+**[VCH Create Command]** `./vic-machine-linux create -t <vcenter_IP> -u <vcenter_username> -p <vcenter_password> --image-store <datastore> --bridge-network <bridge_network> --public-network <network> --registry-ca ca.crt --no-tlsverify --force`
+
+**[Add Registry Path]** Templates > Registries > Add
+
+**[Create Container Path]** Templates > Containers > Create
+
+**[Deploy New OVA Link]** [Deploy New OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/deploy_vic_appliance.html)
+
+**[OVA New Version]** 1.2.0
+
+**[Upgrade OVA Link]** [Upgrade OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/upgrade_appliance.html)
+
+
+Test - Upgrade from OVA 1.1.1 to 1.2.1
+======================================
+**[Deploy Old OVA Link]** [Deploy Old OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.1/vic_vsphere_admin/deploy_vic_appliance.html)
+
+**[OVA Old Version]** 1.1.1
+
+**[Root Cert Path]** Admin > Download Root Cert
+
+**[VCH Create Command]** `./vic-machine-linux create -t <vcenter_IP> -u <vcenter_username> -p <vcenter_password> --image-store <datastore> --bridge-network <bridge_network> --public-network <network> --registry-ca ca.crt --no-tlsverify --force`
+
+**[Add Registry Path]** Templates > Registries > Add
+
+**[Create Container Path]** Templates > Containers > Create
+
+**[Deploy New OVA Link]** [Deploy New OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/deploy_vic_appliance.html)
+
+**[OVA New Version]** 1.2.1
+
+**[Upgrade OVA Link]** [Upgrade OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.2/vic_vsphere_admin/upgrade_appliance.html)
+
+
+Test - Upgrade from OVA 1.1.1 to 1.3.0
+======================================
+**[Deploy Old OVA Link]** [Deploy Old OVA Link](https://vmware.github.io/vic-product/assets/files/html/1.1/vic_vsphere_admin/deploy_vic_appliance.html)
+
+**[OVA Old Version]** 1.1.1
+
+**[Root Cert Path]** Admin > Download Root Cert
+
+**[VCH Create Command]** `./vic-machine-linux create -t <vcenter_IP> -u <vcenter_username> -p <vcenter_password> --image-store <datastore> --bridge-network <bridge_network> --public-network <network> --registry-ca ca.crt --no-tlsverify --force`
+
+**[Add Registry Path]** Templates > Registries > Add
+
+**[Create Container Path]** Templates > Containers > Create
+
+**[Deploy New OVA Link]** [Deploy New OVA Link](TBD - waiting on official docs to link)
+
+**[OVA New Version]** 1.3.0
+
+**[Upgrade OVA Link]** [Upgrade OVA Link](TBD - waiting on official docs to link)
+
+
+Test - Upgrade from OVA 1.1.1 to 1.2.0 to 1.2.1
```

Comment:
Should we add to 1.3 at the end?

Reply:
yes, sure

Reply:
added

Reply:
Upgrade 1.2.0 to 1.3.0. Upgrade 1.2.1 to 1.3.0